nodejs/src/access_token_common.js

```diff
@@ -100,8 +99,8 @@ export class AccessTokenCommon extends ApiCommon {
                     'access_token': this.access_token,
```

Comment:
nit: you could do

```
{
  name: this.name,
  access_token: this.access_token,
  refresh_token: this.refresh_token,
  scopes: this.scopes
};
```

any maybe do without the intermediate output variable? simply pass it as input, I don't see output being used elsewhere in this method.

Reply:
done. also modified other structures in the repo to eliminate quotes on keys.